### PR TITLE
✨ Add rider selection

### PR DESCRIPTION
**Pull request by arassaria**
# Description

Add rider selection

## Type of change

- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -94,6 +95,17 @@ app.get("/api/:collectionName/:id", async (req, res) => {
...
 });
 
+app.get("/api/:collectionName/filter/:key", async (req, res) => {
+  const { collectionName, key } = req.params;
+  try {
+    const data = await getListByKey({ collectionName, key });
+    res.send(data);
+  } catch (e) {
+    console.error(e);
+    res.status(500).send("An unexpected tea pot exploded.");
```
Comment (phlgr) [inline server.js]: Consider using `418`
```suggestion
    res.status(418).send("An unexpected tea pot exploded.");
```

**Review by phlgr — APPROVED** [review]
LGTM 🚀

File: server.js (hunk 1 above)
Comment (arassaria) [reply server.js]: Changed the error text to a normal one. 😉

**Review by Naasgul — APPROVED** [review]
lgtm!

**Review by bwnstck — APPROVED** [review]
aproooved

**Pull request merged**